example/.gitignore

```diff
@@ -1,6 +1,9 @@
 .serverless
 /node_modules/
 
+# ignore ignitor builds
+ignitor
```

Comment:
Can we nest these in the `.serverless` director or somewhere else that's already ignored?